english/marshmallow.md

```diff
@@ -585,7 +585,7 @@ And then you realize that Burning Man is over, so no real vacations :(
 If you eat a lot, you should also [exercise](exercise/exercise.md)
 
 A quick brown fox jumps over the lazy dog!
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.